#!/usr/bin/env python
PACKAGE = "rosneuro_feedback_wheel"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("left_threshold", double_t, 0, "Left threshold", 0.7, 0.5, 1.0)
gen.add("right_threshold", double_t, 0, "Right threshold", 0.7, 0.5, 1.0)

exit(gen.generate(PACKAGE, "rosneuro_feedback_wheel", "ControlWheel"))
